### PR TITLE
chore(query): Update .gitignore

### PR DESCRIPTION
**Pull request by andrewtruong**

### CODE REVIEW

**Review by gtarpenning — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -3,31 +3,20 @@
...
 gha-creds-*.json
-.vscode/launch.json
+.vscode
 .mypy_cache
 .hypothesis
 .env
-yarn-error.log*
-config/.admin.env
-config/.wb_api.env
-weave/tests/zlib.txt
```
Comment (gtarpenning) [inline .gitignore:26]: so much nicer

Comment (gtarpenning) [reply .gitignore:26]: but just to be clear, this is gone?

Comment (gtarpenning) [reply .gitignore:26]: what if i already have an admin.env file in that folder, will this now appear in git?

Comment (andrewtruong) [reply .gitignore:26]: We don't have a `config` dir anymore -- do you need it?

**Pull request merged**